proxy.py

```diff
@@ -11,7 +11,26 @@ def __init__(self):
         super().__init__()
         self.logger = get_nio_logger("ConfigProxy")
 
-    def load_configuration(self, url, apikey):
+    def get_version(self, url_prefix, config_id, apikey):
+        """
+        Retrieves the latest instance configuration version id
+
+        Returns: a configuration vesion id string
```

Comment:
I don't think this returns a string based on line 28, should it? Or should the docs be updated?

Reply:
Ohh you're right. I forgot to update the docs here after updating the API AC